test/distributed/org/apache/cassandra/distributed/upgrade/CompactStorageUpgradeTest.java

```diff
@@ -37,7 +37,7 @@ public void compactStorageColumnDeleteTest() throws Throwable
         new TestCase()
         .nodes(2)
         .nodesToUpgrade(2)
-        .upgradesFrom(v30)
+        .singleUpgrade(v30)
```

Comment:
there are too many tests to try multiple upgrade paths from 3.0... so avoid and just run 3.0 -> trunk

Reply:
upgrading from 3.11 is (a slightly more) recommended upgrade path, not just 3.0

the upgrade tests need to test our supported and recommended upgrade paths, IMHO.

see here: https://lists.apache.org/thread/odk114pfpzpwj81sj2vdzmzt5rgn74wk

Reply:
the function mutates a list, so we could always do

```
.singleUpgrade(v30)
.singleUpgrade(v3x)
```

so we test both

Reply:
updated, we now have

```
INFO  [main] 2022-05-26 11:29:59,284 UpgradeTestBase.java:207 - Adding upgrade of 3.0.28 -> [4.2]
INFO  [main] 2022-05-26 11:29:59,285 UpgradeTestBase.java:207 - Adding upgrade of 3.11.14 -> [4.2]
```

Reply:
wfm. (presuming `[4.2]` becomes whatever current is, so long as the from-to remains something found in the supported_paths)